SEED-platform/seed
# fix titlecase method with apostrophes

#### Any background context you want to provide?
Strings that were being title cased were getting a capital letter after the apostrophe. This is known issue in python's title() method.

#### What's this PR do?
use python's suggestion to fix the title() issue.  Note that this fix is code side only and no data in the database is stored with the weird title case.

#### How should this be manually tested?
Import a file that has an apostrophe in the column title and make sure that the mapping results look right.

#### What are the relevant tickets?
#1257 

#### Screenshots (if appropriate)
#### Definition of Done:
